### ISSUE TITLE
Use the demo to get feedback on usability

### ISSUE THREAD

**Issue by jamalv** [teamforus/me#20]
- [ ] ask random people to try the app, learn where they get confused/stuck
- [ ] improve the demo app based on feedback
- [ ] repeat 🙂

**Comment by maxvisser** [comment]
We can do this in the testing phase. The two native apps probably are alot further in development by then, so we can do usability test with the real apps instead of the phonegapp Me.

**Comment by jamalv** [comment]
related: https://github.com/teamforus/concept-identity/projects

**Comment by maxvisser** [comment]
In my opinion the phonegap should not be developed any further. let rocketminds focus on front-ends of kindpakket 2.0. (time and money is there better spend) And forus should focus on developing two MVP's of the app Me in native android/ios to store kindpakket tokens and personal information to obtain these tokens. Creating the complete vision of the app Me is not realistic for kindpakket 2.0. 

tldr; testing UX design of an app that is a long-term project is not feasible for kindpakket 2.0

action: Should discus this on friday in the kindpakket meeting

**Comment by danrminds** [comment]
I agree here with Jamals approach more, as implementing the design in the native apps will happen only when all functionalities are there. Functionalities also depend on the UX feedback, so will be more cost and time effective to implement feedback changes in the design and implement features before the testing phase.

Yes, discuss it together on Friday.

**Comment by maxvisser** [comment]
me, @danrminds @maartenfv and @jamalv had a discussion about this issue. And decided that the steps discussed here is more about working front - to - back than about the strategic decision to develop the whole me-app for kindpakket 2.0

Our goal with the me app should be to meet the requirements of Zuidhorn and Lung funds. Creating the designs first and testing usability is really valuable.

Risk with this approach are:
Stakeholder think the DEMO app is usable in production. No need to stake any further.

**Comment by maxvisser** [comment]
I think this is superseded by https://github.com/teamforus/research-and-development/issues/84 and will close this issue.
